autosarfactory/autosar_ui.py

```diff
@@ -37,13 +62,31 @@ def __initialize_ui(self):
 
         # set theme
         style = ThemedStyle(self.__root)
+        #print(style.themes)
+#        ['yaru', 'default', 'vista', 'classic', 'scidgreen', 'equilux', 'scidgrey', 'adapta', 'scidpink', 'scidmint', 'plastik', 'alt', 'clearlooks', 'itft1', 'smog', 'clam', 'scidsand', 'kroc', 'radiance', 'black', 'blue', 'arc', 'winxpblue', 'scidblue', 'ubuntu', 'keramik', 'winnative', 'elegance', 'aquativo', 'scidpurple', 'xpnative', 'breeze']
+        #style.theme_use("equilux") #- very slow for some reason
+
         style.theme_use('scidgreen')
-        #or use -->adapta,arc,scidgreen,radiance
 
         # create ui components
         splitter = tk.PanedWindow(orient=tk.VERTICAL)
         top_frame = tk.Frame(splitter)
 
+        menubar = Menu(self.__root)
+        filemenu = Menu(menubar, tearoff=0)
+        menubar.add_cascade(label="Select theme", menu=filemenu)

```

Comment:
I guess, capital 'T' for theme will look more nicer. Still its a personal choice. I am totally fine if you wish to keep it as it is now.

Reply:
Ok.